app/src/main/java/com/github/wanderwise_inc/app/ui/list_itineraries/LikedScreen.kt

```
@@ -82,6 +98,16 @@ fun DisplayLikedItineraries(mapViewModel: MapViewModel, profileViewModel: Profil
                       itinerary.title.contains(searchQuery, ignoreCase = true) ||
                       itinerary.description?.contains(searchQuery, ignoreCase = true) ?: false
                 }
+                .filter { itinerary ->
+                  val price = itinerary.price.toFloat()
+                  price in sliderPositionState.value.start..sliderPositionState.value.endInclusive
```

The return boolean from the operator "in" isnt used so it has no purpose, you need to throw an error or do something if price is not in range

---

yes, it is in my test plans